src/test/java/Homework18.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+    public void playSong(){
+
+        navigateToPage();
+        provideEmail("elena.min@testpro.io");
+    providePassword("te$t$tudent");
+    clickSubmit();
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)